plugins/meson.build

```diff
@@ -1,3 +1,15 @@
+plugin_deps = [
+  gio,
+  giounix,
+  gmodule,
+  gudev,
+  gusb,
+  libarchive,
+  libjsonglib,
+  libxmlb,
+  protobufc,
```

Comment:
maybe add a comment to give us a hint it's not actually a hard dep?

Reply:
k